base/applications/charmap_new/lang/de-DE.rc

```diff
@@ -0,0 +1,22 @@
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
```

Comment:
LANGUAGE LANG_GERMAN, SUBLANG_NEUTRAL

Reply:
Yeah. I already fixed that but g(sh)it reverted it. Sigh.